packages/records/lib/models/records.ts

```diff
@@ -22,6 +22,17 @@ dayjs.extend(utc);
 
 const BATCH_SIZE = 1000;
 
+export async function getRecordsCount({ connectionId, model }: { connectionId: number; model: string }): Promise<Result<number>> {
+    try {
+        const [result] = await db.from(RECORD_COUNTS_TABLE).where({ connection_id: connectionId, model }).select('object_count');
+
+        return Ok(result ? Number(result.object_count) : 0);
+    } catch (_error) {
```

Comment:
note if you are not using the error you can omit entirely the `(err) `

Reply:
I think returning the error here is still useful though, right? Maybe I am misinterpreting

Reply:
I meant the `catch (_error) {` could be replaced with `catch {